mysqldb.bom

```diff
@@ -57,13 +57,17 @@ brooklyn.catalog:
           cat > install-mysql.sh <<END
           #!/usr/bin/env bash
           set -e
+          # workaround for Ubuntu 16.04 problem with AppArmor denying access to a symlinked file
+          echo '/etc/mysql/** lr,' >> /etc/apparmor.d/local/usr.sbin.mysqld
+          sudo service apparmor restart
+          # install mysql 5.6
           export DEBIAN_FRONTEND="noninteractive"
           debconf-set-selections <<< "mysql-server mysql-server/root_password password ${ROOT_PASSWORD}"
           debconf-set-selections <<< "mysql-server mysql-server/root_password_again password ${ROOT_PASSWORD}"
-          # for mysql 5.6
-          sudo add-apt-repository 'deb http://archive.ubuntu.com/ubuntu trusty universe'
-          sudo apt-get update
-          sudo apt install -y mysql-server-5.6 mysql-client-5.6 augeas-tools
+          apt-get install -y software-properties-common
```

Comment:
move that after line 69

Reply:
It needs to be done here as it brings in `add-apt-repository` used on the following line.  That in turn means the subsequent `apt-get update` loads the definitions of the 5.6 version of mysql so they can be installed.

Reply:
I'm seeing a problem with line 67 if `apt-get update` is not executed before, possibly it is only on my vagrant machine, especially as you are using AMIs for that.
looks good